main/guides/getting-started/README.md

```diff
@@ -1,163 +1,199 @@
+If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+
+Also, check that you are on a
+[supported platform](#platform-linux-shell-or-equivalent) and
+not native Windows.
+:::
+
+## Start a local Agoric blockchain
+
+To run a local Agoric blockchain with [docker-compose](https://docs.docker.com/compose/):
+
+```sh
+yarn start:docker
 ```
 
-Cloning and installing the Agoric SDK can take a while. Please be patient.
+::: tip Note: large docker image
 
-## Install NPM Dependencies
+The docker image used here is several gigabytes.
+The fist time you pull it may take several minutes.
 
-```shell
-yarn install
+:::
+
+Look at the logs from the blockchain until it is steadily making blocks:
+
+```sh
+yarn docker:logs
 ```
 
-**Note:** If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+## Deploy the Contract and Start the UI
 
-## Build Packages
+Let's start it up:
 
-```shell
-yarn build
+```sh
+yarn start
 ```
 
-**Note:** If this `yarn build` step fails, check that you are on a
-[supported platform](#platform-linux-shell-or-equivalent) and
-not native Windows.
+::: tip TODO: integrated `yarn start` script
 
-## Build the Cosmic Swingset Package
+For now, use the [Agoric Gov Proposal Builder](https://cosgov.org/)
+for deployment:
 
-```shell
-(cd packages/cosmic-swingset && make)
+1. Make the contract and proposal bundles.
+
+```sh
+(cd contract; yarn build:proposal)
 ```
 
-## Install Agoric CLI
+Note the long `b1-xxx.json` bundle filenames
+as well as `start-game1-permit.json`
+and `start-game1.js`.
 
-Use `yarn link-cli` to install the Agoric CLI (Command Line Interface) in a convenient place of your choosing such as:
+2. Use the [Install Bundle](https://cosgov.org/?msgType=installBundle&network=local) tab to install the 2 bundles.
+   It will likely say **insufficient balance**.
+   To get enough IST:
 
-```shell
-yarn link-cli ~/bin/agoric
+```sh
+yarn docker:make mint4k
 ```
 
-or:
+2. Get ready to vote. To query the status of proposals, use
 
-```shell
-sudo yarn link-cli /usr/local/bin/agoric
+```sh
+yarn docker:make gov-q
 ```
 
-**Note:** Run `echo $PATH` to see directories in your current path, separated by colons. These are good candidates for where to have `yarn link-cli` place the executable.
+Then, don't execute this command, but get it ready:
 
-::: tip Troubleshooting "command not found"
-Watch:
+```sh
+yarn docker:make vote PROPOSAL=N
+```
 
-- [Linux add to \$PATH: Fix "command not found" error (Linux & Mac)](https://www.youtube.com/watch?v=gkqsLRDnqlA) 6:19 Mar 2018.
-  :::
+3. Use the [CoreEval Proposal](https://cosgov.org/?msgType=coreEvalProposal&network=local) tab to make a proposal to
```

Comment:
These steps accurately capture the current flow, but it would be nice if these actions (faucet, install bundle, core eval, vote) are obscured behind a single command. These concepts can be introduced later.

Reply:
> I wonder how many of the yarn scripts from contracts/package.json to make available from the top-level.

I think exposing `docker:logs` and `docker:exec` would be useful here.
  - "To follow logs in a separate terminal, enter `yarn docker:logs`." 
  - "The docker container is running a fork of the latest version of mainnet. Here you can run any agd commands with governance keys, or leverage included scripts like printAllKeys and voteLatestProposalAndWait"

Reply:
> it would be nice if these actions (faucet, install bundle, core eval, vote) are obscured behind a single command

done.

> Here you can run any agd commands with governance keys ...

I prefer to leave that out of this flow. What I have currently is:

> This `start:contract` script will do a number of things that we will cover in more detail later: